test/e2e/parallel/nstemplatetier_test.go

```diff
@@ -378,6 +379,41 @@ func TestFeatureToggles(t *testing.T) {
 	})
 }
 
+func TestTierTemplateRevision(t *testing.T) {
```

Comment:
could you please add `// given`, `// when` and `// then` comments - for better reading

Reply:
done in https://github.com/codeready-toolchain/toolchain-e2e/pull/1061/commits/7d3484356019f094380984c9f819bacb5bbf1695 , thank you 👍